package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/eslint-plugin-node",
-  "version": "1.2.2",
```

Comment:
Would you mind removing this from the PR? In order to get Travis to publish this correctly I'd like to create the version bump and tag after this has been merged.